### PR TITLE
Mail messages

### PR DESCRIPTION
**Pull request by marceloschmidt**

### CODE REVIEW

File: packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.coffee [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+		user = Meteor.user()
+		name = user.name
+		email = user.emails?[0]?.address
+
+		if data.language isnt 'en'
+			localeFn = Meteor.call 'loadLocale', data.language
+			if localeFn
+				Function(localeFn)()
+				moment.locale(data.language)
```
Comment (rodrigok) [inline packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.coffee]: Here are you changing the locale forever?
What happens if you send a mail in PT then another one in EN?

Comment (marceloschmidt) [inline packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.coffee]: Right! I should have used `localMoment = moment().locale(...)`. I'll fix that.

**Comment by rodrigok** [comment]
@marceloschmidt prints? :smile_cat:

**Comment by sampaiodiego** [comment]
is it intended to an user from any access level (not only admins) be able to send the conversation by email?

**Comment by marceloschmidt** [comment]
I don't know... what do you guys think? Should we have a permission for that? If we need a role, we need an UI for adding a specific user to roles...

**Comment by marceloschmidt** [comment]
Should we create roles for that?

**Comment by sampaiodiego** [comment]
A *permission* at least.. So admins can create their own roles with that permission.

**Comment by marceloschmidt** [comment]
mailmessages

**Comment by marceloschmidt** [comment]
Added `mail-messages` permission to admins only, for now.

**Comment by rodrigok** [comment]
LGTM

**Comment by sampaiodiego** [comment]
lgtm

**Pull request merged**